resources/content/schema/admin/admin-auth.json

```diff
@@ -58,10 +58,15 @@
         "externalHandler.retries" : "cru",
         "externalHandler.timeoutMillis" : "cru",
 
+        "githubconfig.enabled": "cr",
```

Comment:
move down to be alphabetical and also next to the existing "githubconfig": "cru".  I'm guess this was a rebase conflict that moved this up